distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -529,8 +529,8 @@ type Decimal
        Convert this to an `Integer`.
 
        > Example
-         Convert `Decimal` 2345 to an `Integer`.
-             d = Decimal.new "2345"
+         Convert `Decimal` 2345.6 to an `Integer`.
+             d = Decimal.new "2345.6"
```

Comment:
So the behaviour is `truncate`? IMO it would be good to specify this clearly in the description.

Reply:
Documented and added test.